dynamodb/src/main/scala/awscala/dynamodbv2/Table.scala

```diff
@@ -51,6 +53,21 @@ case class Table(
   def put(hashPK: Any, attributes: (String, Any)*)(implicit dynamoDB: DynamoDB): Unit = putItem(hashPK, attributes: _*)
   def put(hashPK: Any, rangePK: Any, attributes: (String, Any)*)(implicit dynamoDB: DynamoDB): Unit = putItem(hashPK, rangePK, attributes: _*)
 
+  def put[E <: AnyRef](hashPK: Any, entity: E)(implicit dynamoDB: DynamoDB, c: ClassTag[E]): Unit =
+    dynamoDB.put(this, hashPK, getAttrValuesToList(entity): _*)
+
+  def put[E <: AnyRef](hashPK: Any, rangePK: AnyRef, entity: E)(implicit dynamoDB: DynamoDB,c: ClassTag[E]): Unit =
+    dynamoDB.put(this, hashPK, rangePK, getAttrValuesToList(entity): _*)
+
+  private def getAttrValuesToList(entity: AnyRef) = {
+    val fields = entity.getClass.getDeclaredFields.map(_.getName).toList
```

Comment:
Your code still accesses private fields. Use `getDeclaredMethods` and check their modifiers.

Reply:
hey @seratch I refactored that one more time. Since this is Scala 2.13 I decided to use productElementNames. 
btw. hashPK and rangePK have been moved into case class definition. I guess it makes it more consistent now.